src/UI/Text.elm

```diff
@@ -60,9 +61,9 @@ A text can be created and rendered as in the following pipeline:
 @docs withColor
 
 
-# Ellipsis
+# Overflow
 
-@docs setEllipsis
+@docs withOverflow
```

Comment:
Can you add the other related functions here as well? (ellipsize, ellipsizeWithTooltip, and wrap)